server/controllers/notification.js

```diff
@@ -0,0 +1,65 @@
+const Notification = require("../models/Notifications");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /notifications/create
+// @desc create new notification
+// @access Public
+exports.createNotification = asyncHandler(async (req, res, next) => {
+    const notification = new Notification({
+        type: req.body.type,
+        title: req.body.title,
+        description: req.body.description,
+        read: req.body.read,
+        date: req.body.date
+    });
+    const newNotification = await notification.save()
+    res.status(200).json({
+        success: {
+            notification: newNotification,
+        }
+    })
+});
+
+// @route POST /notifications/read
+// @desc mark notification as read
+// @access Public
+exports.readNotification = asyncHandler(async (req, res, next) => {
+    const notification = await Notification.findById(req.body.id);
+    if (!notification) {
```

Comment:
Nitpick! You could pass `notificationId` as a request param and use a dynamic route with a `PATCH` request.